third_party/2and3/requests/__init__.pyi

```diff
@@ -9,32 +9,20 @@ from . import exceptions
 from . import packages
 import logging
 
+from .models import Request, Response, PreparedRequest
```

Comment:
You need to write these as `from . models import Request as Request` to mark them as re-exported.